### PR TITLE
Xosonc

### PR DESCRIPTION
**Pull request by anisimkov**

### CODE REVIEW

File: config/setup/xutil.adb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 --                                                                          --
 --                                 B o d y                                  --
 --                                                                          --
---          Copyright (C) 1992-2012, Free Software Foundation, Inc.         --
+--          Copyright (C) 1992-2018, Free Software Foundation, Inc.         --
```
Comment (TurboGit) [inline config/setup/xutil.adb]: Why? There is no change in this file.

File: config/setup/xutil.ads [hunk 2]
```diff
@@ -6,7 +6,7 @@
 --                                                                          --
 --                                 S p e c                                  --
 --                                                                          --
---          Copyright (C) 1992-2012, Free Software Foundation, Inc.         --
+--          Copyright (C) 1992-2018, Free Software Foundation, Inc.         --
```
Comment (TurboGit) [inline config/setup/xutil.ads]: Why? There is no change in this file.

Comment (anisimkov) [reply config/setup/xutil.ads]: I just copied it from gnat sources.

File: config/setup/xutil.adb (hunk 1 above)
Comment (anisimkov) [reply config/setup/xutil.adb]: I just copied it from gnat sources

Comment (anisimkov) [reply config/setup/xutil.adb]: And I think it is a bit better to update then to do not update.

File: config/setup/xutil.ads (hunk 2 above)
Comment (TurboGit) [reply config/setup/xutil.ads]: Ok,  but there is no change here so the update is wrong to me. Some projects do update the copyright notice each start of year, AWS does not.

**Comment by anisimkov** [comment]
Ok, I removed xutil.ad? change from commit

**Comment by anisimkov** [comment]
<<Indeed the constant is not currently used in AWS, but it is harmless? Or does it introduce some conflicts?>> No conflicts. Ok abandon this commit if you don't like it. It was just experimenting for me. Not really need.

**Pull request merged**